template/partial/safety_plan_worksheet/concern_1.html

```diff
@@ -0,0 +1,16 @@
+{% set ribbon_data = { color: 'red', text: 'Physical Safety: Concerns'} %}
+{% include 'partial/ribbon.html' with ribbon_data only %}
+<p>Have you anticipated where and in which ways you might come into contact with the perpetrator and perpetrator’s friends/family? If you were to come into contact with the perpetrator and perpetrator’s family/friends, what specific things could you do that might help you feel safe in that situation?
```

Comment:
@AkankshaBodhankar According to HTML DOM tree structure this should not be indented to four spaces because There is no parent to these tags. 
<img width="1082" alt="screen shot 2017-06-14 at 11 57 24" src="https://user-images.githubusercontent.com/13470643/27118392-b30734ac-50f8-11e7-86fb-9152f9f274c8.png">